src/App.js

```diff
@@ -383,26 +383,28 @@ class App extends Component {
     const shouldConfigure = !this.state.serverConfig.mqttBrokerUrl || !this.state.serverConfig.domoticzUrl;
     const view = this.renderCurrentView(shouldConfigure ? View.SERVER_SETTINGS : undefined);
     const currentView = this.state.currentView;
+    const cssIcon = this.state.theme.icon;
+    const cssIconSelected = this.state.theme.iconSelected;
```

Comment:
In the rest of the Reacticz codebase, we generally define a style object here, eg:
```
const menuIconStyle = {
  fill: this.state.theme.icon
}
const menuIconSelectedStyle = {
  fill: this.state.theme.iconSelected
}
```

Then in the return statement:
```
...
<svg className='icon' style={currentView === View.DASHBOARD ? menuIconSelectedStyle : menuIconStyle}>
...
```